header-footer-grid/Core/Builder/Abstract_Builder.php

```diff
@@ -449,6 +451,67 @@ public function define_row_settings( $row_id ) {
 				'default'               => $default_colors['text'],
 			]
 		);
+
+		SettingsManager::get_instance()->add(
+			[
+				'id'                    => self::BOTTOM_BORDER,
+				'group'                 => $row_setting_id,
+				'tab'                   => SettingsManager::TAB_STYLE,
+				'section'               => $row_setting_id,
+				'label'                 => __( 'Divider width', 'neve' ),
```

Comment:
Arina, to avoid adding new strings for translation, I would use only `Border Width` which we already use in other controls.